tests/client.c

```diff
@@ -492,34 +435,41 @@ main(int argc, const char **argv)
   int result = 1;
   if(argc <= 2) {
     fprintf(stderr,
-            "usage: %s hostname path\n\n"
-            "Connect to a host via HTTPS on port 443, make a request for the\n"
-            "given path, and emit response to stdout.\n",
+            "usage: %s hostname port path\n\n"
+            "Connect to a host via HTTPS on the provided port, make a request "
+            "for the\n"
+            "given path, and emit response to stdout (three times).\n",
             argv[0]);
     return 1;
   }
   const char *hostname = argv[1];
-  const char *path = argv[2];
+  const char *port = argv[2];
+  const char *path = argv[3];
 
   struct rustls_client_config_builder *config_builder =
     rustls_client_config_builder_new();
   const struct rustls_client_config *client_config = NULL;
-  const struct rustls_slice_bytes alpn_http11 = { (const uint8_t *)"http/1.1", 8 };
+  const struct rustls_slice_bytes alpn_http11 = { (const uint8_t *)"http/1.1",
+                                                  8 };
```

Comment:
nit: If you're OK with requiring C99, designated initializers are very good for readability
```suggestion
  const struct rustls_slice_bytes alpn_http11 = { 
      .data = (const uint8_t *)"http/1.1",
      .len = 8,
  };
```

Reply:
I decided not to use the designated initializers for now, but to set fields by hand.